readme.md

```diff
@@ -389,6 +389,19 @@ const usePlainStore = create(devtools(store))
 const useReduxStore = create(devtools(redux(reducer, initialState)))
 ```
 
+One redux devtools connection for multiple stores
+```jsx
+import { devtools } from 'zustand/middleware'
+
+// Usage with a plain action store, it will log actions as "setState"
+const usePlainStore1 = create(devtools(store), { name, store: storeName1 })
```

Comment:
Do we have any other naming idea for `store` property?

Reply:
Afaiu, how about instead of having `{ name, store }` we have `{ name, instanceName }`? So the implementation would have bits like `extension.connect({ name: options.instanceName ?? options.name })` and `if (options.instanceName !== undefined) currentStates[options.name] = ....`

Reply:
I just thought, we already have `type` to name actions, then we could rename `store` to `typePrefix`. Because indeed this property is added as prefix to `type`